.eslintrc.yml

```diff
@@ -49,7 +49,7 @@ rules:
     - error
     - 4
     - SwitchCase: 1
-      MemberExpression: 0
+      MemberExpression: 1
```

Comment:
Multi-line statements would get indented on the same level with MemberExpression 0, I think have an indent makes it more readable.

Reply:
I was bashing my head against the wall yesterday with exactly the same problem. Thanks for figuring it out.